File: playground/java/src/main/java/playground/order/OrderService.java
package playground.order;

import java.util.List;

public final class OrderService {
    public Order createSampleOrder() {
        return new Order(
            "Linus",
            List.of(
                new OrderItem("SSD", 1, 89.0),
                new OrderItem("Bracket", 2, 6.5)
            )
        );
    }
}

File: playground/kotlin/src/main/kotlin/playground/order/OrderService.kt
package playground.order

fun sampleOrder(): Order = Order(
    customer = "JetBrains",
    items = listOf(
        OrderItem(name = "Keyboard", quantity = 1, price = 120.0),
        OrderItem(name = "Keycap", quantity = 4, price = 3.5)
    )
)
